templates/collection.nomad

```diff
@@ -92,9 +92,39 @@ job "collection-${name}" {
           pg = 5432
         }
       }
-      env {
-        POSTGRES_USER = "snoop"
-        POSTGRES_DATABASE = "snoop"
+      template {
+        data = <<EOF
+          POSTGRES_USER = "snoop"
+          POSTGRES_DATABASE = "snoop"
+          {{- with secret "liquid/collections/${name}/snoop.postgres" }}
+            POSTGRES_PASSWORD = {{.Data.secret_key}}
+          {{- end }}
+        EOF
+        destination = "local/postgres.env"
+        env = true
+      }
+      template {
+        data = <<-EOF
+        #!/bin/sh
+        set -ex
+        pwd
+        date
+        if grep -Fq "$host all all all trust" $PGDATA/pg_hba.conf
+        then
+          psql -U snoop -c "ALTER USER snoop password '$POSTGRES_PASSWORD'"
```

Comment:
This, combined with `set -x`, will write the db password in nomad logs. You can wrap this line in parantheses and `set +x` will only apply within:
```shell
(
set +x
psql -U snoop -c "ALTER USER snoop password '$POSTGRES_PASSWORD'"
)
```

Reply:
Yeah that was my last missing piece, I wanted to ask you guys about that. The password will also be logged during the startup code, I think.
```
        if [ -z "$SNOOP_DB" ] \
                || [ -z "$SNOOP_TIKA_URL" ] \
                || [ -z "$SNOOP_ES_URL" ] \
                || [ -z "$SNOOP_AMQP_URL" ]; then
```
I'll change it there as well, okay?

Reply:
Yeah, we want no passwords leaking into the logs.